packages/nouns-webapp/src/pages/Vote/index.tsx

```diff
@@ -293,24 +296,41 @@ const VotePage = ({
             </Col>
           </Row>
         )}
+
+        <p
+          onClick={() => setIsDelegateView(!isDelegateView)}
+          className={classes.toggleDelegateVoteView}
+        >
+          {isDelegateView ? (
+            <Trans>Switch to Noun view</Trans>
+          ) : (
+            <Trans>Switch to delegate view</Trans>
+          )}
```

Comment:
Oddly, this text is not showing up in the [preview](https://deploy-preview-475--nouns-testnet.netlify.app/vote/83), but does appear locally. Are you seeing the same behavior?